skills/common/process/batch/runtime/powershell/Scripts/Invoke-BatchOrchestrator.ps1

```
@@ -259,6 +372,27 @@ function Invoke-BatchOrchestration {
+                        if ($issue.State -eq "ORPHANED") {
+                            $tempConfig = New-SubAgentConfig -MaxRetries $MaxRetries
+                            $tempState = New-SubAgentState -IssueId $issueId -Config $tempConfig
+                            $tempState.RetryCount = $issue.RetryCount
+                            $retryCheck = Test-SubAgentRetryable -SubAgentState $tempState -ErrorCategory "transient"
+                            if ($retryCheck.Retryable) {
+                                Write-BatchLog "Issue $issueId: Recovering from ORPHANED (attempt $($issue.RetryCount + 1)/$($MaxRetries))" "WARN"
+                                $issue.LastError = "Recovering from orphaned state"
+                                $issue.SubAgentProcessId = $null
+                                Set-IssueStateTransition -IssueState $issue -ToState "WAITING_FOR_SUBAGENT" -BatchId $BatchId -Reason "Recovery: retry eligible"
+                            } else {
+                                Write-BatchLog "Issue $issueId: ORPHANED recovery exhausted ($($retryCheck.Reason))" "ERROR"
```

PSScriptAnalyzer (1.25.0)

[error] Variable reference is not valid. ':' was not followed by a valid variable name character. Consider using ${} to delimit the name.

(InvalidVariableReferenceWithDrive)